app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import shutil
+
+
+def copy_file(source_file: str, destination_file: str) -> None:
+    try:
+        shutil.copy(source_file, destination_file)
+        print(f"File copied from {source_file} to {destination_file}")
```

Comment:
Code Style Issue: It is recommended to use one style of quotes in your code. Double quotes are preferable.